maestro-cli/src/main/java/maestro/cli/device/DeviceService.kt

```diff
@@ -113,31 +117,36 @@ object DeviceService {
         }
     }
 
-    fun listConnectedDevices(): List<Device.Connected> {
-        return listDevices()
+    fun listConnectedDevices(includeWeb: Boolean = false): List<Device.Connected> {
```

Comment:
Why a boolean here? Shouldn't we by default list web devices?

Reply:
That would be rather confusing for mobile engineers because then we will always try to run the test in a browser if no mobile devices are connected